src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -102,14 +110,21 @@ var Stream = React.createClass({
   },
 
   fetchTags() {
+    StreamTagStore.reset();
```

Comment:
Well, I suppose we could make `loadTags` perform a reset instead of doing it here.